Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,13 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./css/main.css";
+import Display from "./components/DisplayComponents/Display";
+import ButtonsGroup from "./components/ButtonComponents/ButtonsGroup";
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="chassis">
+      <Display />
+      <ButtonsGroup />
```

Comment:
I much prefer this structure where you can have ButtonsGroup return all of your button components without having component-soup all in App.js